Runner/Program.cs

```diff
@@ -163,7 +162,33 @@ private static GitHubClient CreateClient(GitHubConfiguration configuration)
             return client;
         }
 
-        private static ServiceProvider BuildDependencyInjection(IConfiguration config) => new ServiceCollection()
+        private static ServiceProvider BuildDependencyInjection(IConfiguration config)
+        {
+            var provider = new ServiceCollection();
```

Comment:
You are creating `ServiceCollection` twice which is not necessary here, but I think this implementation should be more generic anyway,

Reply:
Fixed but feractored